### DISCUSSION TITLE
moving lists of appbar items

### DISCUSSION THREAD

**Discussion by 7576457** [flet-dev/flet#1797] [Q&A]
### Question

![2023-09-04](https://github.com/flet-dev/flet/assets/105982046/16487473-6287-4be5-94a5-a14c5ffe5946)
Can I move the elements of the "actions" application panel to the "leading" position?
If I try to pass the list of elements to "leading", I get t![Снимок экрана от 2023-09-04 15-51-25](https://github.com/flet-dev/flet/assets/105982046/ae3ca2f9-d5e3-4e5e-89d9-8d7891349c58)

### ------------------------------------------------------

- [x] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
Could you provide a code to reproduce?
